rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -61,7 +61,7 @@ spec:
               Tekton controllers on cluster {{ $labels.source_cluster }} have restarted {{ $value }} times recently.
             alert_team_handle: <!subteam^S04PYECHCCU>
             team: pipelines
-            runbook_url: TBD
+            runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/core-pipeline-controller-repeated-restarts.md
```

Comment:
I can't reach this link.

Reply:
oops I meant to put another of those clarifying comments like with the other additions ... will address in a follow up push

Reply:
comment added